platform/platform-app/src/main/resources/features.xml

```diff
@@ -53,13 +66,6 @@
         <!--</bundle>-->
     <!--</feature>-->
 
-    <feature name='platform-email' install="automatic" version='${project.version}' start-level='50'>
```

Comment:
This feature was moved to the bottom of this file because it was originally added in the camel features section, but it is ddf-specific.